### ISSUE TITLE
tbg multiple openPMD output causes strange naming 

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#4409]
When calling openPMD-api multiple times within a `*.cfg` using `tbg`, the newly added `--openPMD.infix` is escaped from `'_%T'` to `\'_%T\'`. This not only leads to the single quotation marks being written to the output files but to additional double quotation marks around the entire files. 

The naming is changed from `simData_0.bp` to `"simDataEfield'_0'.bp"`.

The double openPMD-api call is given below:

```bash

TBG_openPMD_en="--openPMD.period 20000                         \
            --openPMD.file simDataElectrons                         \
            --openPMD.source en_all                        \
            --openPMD.ext bp                               \
            --openPMD.infix '_%T'                          "
#            --openPMD.dataPreparationStrategy mappedMemory "


TBG_openPMD_E="--openPMD.period 5000                         \
            --openPMD.file simDataEfield                         \
            --openPMD.source E                           \
            --openPMD.range 384,:,:                           \
            --openPMD.ext bp                               \
            --openPMD.infix '_%T'                          "
#            --openPMD.dataPreparationStrategy mappedMemory "


TBG_checkpoints=" --checkpoint.period 30000 "
#                  --checkpoint.openPMD.dataPreparationStrategy mappedMemory   "



TBG_plugins="!TBG_openPMD_en     !TBG_openPMD_E              \
             !TBG_checkpoints              \
             !TBG_sumEnergy                \
             !TBG_en_histogram             \
             !TBG_e_histogram              \
              --e_macroParticlesCount.period 100 \
              --en_macroParticlesCount.period 100"
              
```

After speaking with @psychocoderHPC *offline*, I will test removing the single quotation marks in the `*.cfg` and will remove the escape symbols in the `submit.start`.

**Comment by steindev** [comment]
@PrometheusPi Any updates here?

**Comment by PrometheusPi** [comment]
@steindev Thanks for reminding me. I will have a look at it again.

**Comment by steindev** [comment]
How soon? 😉 😂

**Comment by PrometheusPi** [comment]
later 😜

**Comment by psychocoderHPC** [comment]
solved by #4905
